src/java/org/apache/cassandra/tools/nodetool/Status.java

```diff
@@ -170,24 +215,134 @@ private void addNode(String endpoint, Float owns, HostStatWithPort hostStat, int
         strOwns = owns != null && hasEffectiveOwns ? new DecimalFormat("##0.0%").format(owns) : "?";
         hostID = hostIDMap.get(endpoint);
 
-        try
-        {
+        try {
             rack = epSnitchInfo.getRack(endpoint);
-        }
-        catch (UnknownHostException e)
-        {
+        } catch (UnknownHostException e) {
             throw new RuntimeException(e);
         }
 
-        epDns = hostStat.ipOrDns(printPort);
         if (isTokenPerNode)
-        {
-            tableBuilder.add(statusAndState, epDns, load, strOwns, hostID, hostStat.token, rack);
-        }
+            return List.of(addressAndPort, statusAndState, epDns, load, strOwns, hostID, hostStat.token, rack);
         else
-        {
-            tableBuilder.add(statusAndState, epDns, load, String.valueOf(size), strOwns, hostID, rack);
+            return List.of(addressAndPort, statusAndState, epDns, load, String.valueOf(size), strOwns, hostID, rack);
+    }
+
+    // To check for descending order
+    private Boolean desc() {
```

Comment:
lease be sure that opening  braces are on the new line. We have a codestyle like that across the project we try to keep consistent everywhere.

https://cassandra.apache.org/_/development/code_style.html

See "Formatting" section.

Reply:
I see that we need to follow this format. I will imporve the code with the suggested codestyle